dspace-oai/src/main/java/org/dspace/xoai/app/XOAI.java

```diff
@@ -288,11 +289,20 @@ private int index(Iterator<Item> iterator) throws DSpaceSolrIndexerException {
                     log.error(ex.getMessage(), ex);
                 }
                 i++;
-                if (i % 100 == 0)
+                if (i % 1000 == 0) {
+                    System.out.println(i + " items prepared so far...");
+                }
+                if (i % 10000 == 0) {
                     System.out.println(i + " items imported so far...");
+                    server.add(list);
+                    server.commit(true, true);
```

Comment:
I'm curious why we switched from `server.commit();` to `server.commit(true, true);`?  Don't they do the same thing?  It looks like a default `commit()` may call `commit(true, true)` based on my reading of these Javadocs: https://lucene.apache.org/solr/5_5_0/solr-solrj/org/apache/solr/client/solrj/SolrClient.html#commit()

Reply:
It was one of many things I tried in my testing process before I found the solution.  I'm not sure the change would be worth another test cycle.